e2e/override-globals/__tests__/index.js

```diff
@@ -7,8 +7,6 @@
 
 'use strict';
 
-import _unusedRequireOverridingPromise from '..';
```

Comment:
this should remain

Reply:
can add a comment above it being unused is on purpose? shouldn't be needed as it's in the name, tho